core/src/event/message.rs

```diff
@@ -5,5 +5,6 @@ use serde_json::Value;
 pub struct EventMessage {
     pub event_name: String,
     pub event_data: Value,
+    pub event_topic: String,
```

Comment:
i was wondering if event_topic could stay as Signature i think it has a serde on it and should then work on create_chunk_message_raw and create_chunk_message_json (these are both the places it constructs ready to send)